src/uct/ib/configure.m4

```diff
@@ -152,7 +152,6 @@ AS_IF([test "x$with_ib" = "xyes"],
                        AC_CHECK_DECLS([
                            mlx5dv_init_obj,
                            mlx5dv_create_qp,
-                           mlx5dv_is_supported,
```

Comment:
why removed?

Reply:
it is not needed anymore, we don't check `HAVE_DECL_MLX5DV_IS_SUPPORTED` anymore

Reply:
but if mlx5dv_is_supported is not found, we would disable mlx5dv altogether

Reply:
yes, valid point.
fixed